ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobs.tsx

```diff
@@ -80,6 +80,7 @@ function ReportJobs({ reportId }: RunHistoryProps) {
 
     const handleChange = (checked: boolean) => {
         setShowOnlyMyJobs(checked);
+        setPage(1);
```

Comment:
We don't have to do this since the state management isn't that complex on this page, but I'd imagine a reducer would help here considering the side effects of state changes. I think we should definitely make a test for side effects like this in a follow-up PR though